pkg/diagnose/ports/ports.go

```diff
@@ -40,6 +41,11 @@ func DiagnosePortSuite() []diagnosis.Diagnosis {
 
 	var diagnoses []diagnosis.Diagnosis
 	for _, key := range pkgconfigsetup.Datadog().AllKeysLowercased() {
+		// on windows, we skip the ports used by apm agent and process agent because the core agent does not have permissions to retrieve proc name
+		if runtime.GOOS == "windows" && (strings.HasPrefix(key, "apm_config") || strings.HasPrefix(key, "process_config")) {
```

Comment:
Sorry to bring it up again. Privilege is required to retrieve process full command line but process name retrieval should be allowed (my previous comment was not entirely accurate). Of course it depends what API is trying to get it. Many types of API may fail actually. One which should work is [ZwQuerySystemInformation](https://pkg.go.dev/golang.org/x/sys/windows#NtQuerySystemInformation) with SystemProcessIdInformation.

Reply:
done!